Framework/Lumberjack/CocoaLumberjack.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CocoaLumberjack.h
+//  CocoaLumberjack
+//
+//  Created by Andrew Mackenzie-Ross on 3/02/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for CocoaLumberjack.
+FOUNDATION_EXPORT double CocoaLumberjackVersionNumber;
+
+//! Project version string for CocoaLumberjack.
+FOUNDATION_EXPORT const unsigned char CocoaLumberjackVersionString[];
```

Comment:
Markdown caught my `<>` brackets. The difference is this file has `#import <CocoaLumberjack/_HEADER_>` imports while the other file has `#import "_HEADER_"` imports.